kolibri_explore_plugin/assets/src/customApps.js

```diff
@@ -40,6 +40,7 @@ export const CustomChannelApps = {
   '92e96efc082e5c62b0aac3847bdcdb33': 'staff-playlist',
   '2f95235c3709511fa12d007f31ed6a7b': 'steam',
   '1520f018610256549c98ca0140cceebe': 'virtual-field-trips',
+  eb4373b5da054c07879d0c969dc1976a: 'virtual-science-teachers',
```

Comment:
```suggestion
  'eb4373b5da054c07879d0c969dc1976a': 'virtual-science-teachers',
```

Reply:
Thanks for the check, there is some fun reason why ids starting with a character rather than a number do not need the quotes. See other examples in the same file.